dateutil/rrule.py

```diff
@@ -707,7 +707,7 @@ def __str__(self):
             parts.append('INTERVAL=' + str(self._interval))
 
         if self._wkst:
-            parts.append('WKST=' + str(self._wkst))
+            parts.append('WKST=' + repr(weekday(self._wkst))[0:2])
```

Comment:
Ah, I see what's going on. I actually meant that you'd use `MO(None)`, but I didn't really look closely enough, I didn't realize that `_wkst` was storing an integer and not a `weekday`  object.

I think the reason it's split up like that just below is because for `byweekday` you can use weekday offsets like `SU(1)`, and to conform to the RFC, that has to print as `+1SU`. Since `_wkst` can't have that offset parameter, it's not a concern.

No worries, though, seems fine as is.